### DISCUSSION TITLE
انتخاب تاریخ فارسی و ثبت میلادی در دیتابیس

### DISCUSSION THREAD

**Discussion by mehtism** [majidh1/JalaliDatePicker#61] [Question]
سلام. امکانش هست که بعد از انتخاب تاریخ توسط کاربر و قبل از سابمیت دیتا به میلادی تغییر بکنه؟؟

با تشکر

**Comment by majidh1** [comment]
سلام
این مورد باید سمت سرور کنترل بشه و جایی که باید دیتا تبدیل بشه عمل تبدیل را انجام دهید
اگر سمت کلاینت قرار به انجام باشه بهتره یک hiddenField داشته باشید که با تغییر تاریخ شمسی تاریخ میلادی رو محاسبه کنه و توی سرور از مقدار اون استفاده کنید.
